modules/virt-creating-offline-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-creating-offline-vm-snapshot-cli_{context}"]
+= Creating an offline virtual machine snapshot in the CLI
+
+You can create a virtual machine (VM) snapshot for an offline VM by creating a `VirtualMachineSnapshot` object.
+
+.Prerequisites
+
+* Ensure that the persistent volume claims (PVCs) are in a storage class that supports Container Storage Interface (CSI) volume snapshots.
+* Install the OpenShift CLI (`oc`).
+* Power down the VM for which you want to create a snapshot.
+
+.Procedure
+
+. Create a YAML file to define a `VirtualMachineSnapshot` object that specifies the name of the new `VirtualMachineSnapshot` and the name of the source VM.
++
+For example:
++
+[source,yaml]
+
+----
+apiVersion: snapshot.kubevirt.io/v1alpha1
+kind: VirtualMachineSnapshot
+metadata:
+  name: my-vmsnapshot <1>
+spec:
+  source:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: my-vm <2>
+----
+<1> The name of the new `VirtualMachineSnapshot` object.
+<2> The name of the source VM.
+
+. Create the `VirtualMachineSnapshot` resource. The snapshot controller creates a `VirtualMachineSnapshotContent` object, binds it to the `VirtualMachineSnapshot` and updates the `status` and `readyToUse` fields
+of the `VirtualMachineSnapshot` object.
++
+[source,terminal]
+----
+$ oc create -f <my-vmsnapshot>.yaml
+----
+
+.Verification steps
```

Comment:
Just mentioning that I wasn't aware of this option! I was going to say that we don't do this, but I see it elsewhere in the repo and in the mod docs reference guide as an optional section. Pretty neat, I might need to consider trying this!